Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs

```diff
@@ -63,6 +64,21 @@ public virtual void SetPosition(bool active, Vector3 position)
         }
 
         /// <summary>
+        /// The SetColor method is used to set the clor for the direction indicator to match the custom pointer cursor.
+        /// </summary>
+        /// <param name="color">The color to assign.</param>
+        public virtual void SetColor(Color color)
+        {
+            List<Renderer> renderers = new List<Renderer>();
+            renderers.Add(GetComponent<Renderer>());
+            GetComponentsInChildren<Renderer>(renderers);
```

Comment:
@bddckr well if this is true then I don't need to use List<T> as well. 
I just need to store them in an array instead.